NStackSDK/NStackSDK/Classes/Translations/TranslationManager.swift

```diff
@@ -471,6 +496,10 @@ public class TranslationManager {
     
     /// The URL used to persist downloaded translations.
     var translationsFileUrl: URL? {
-        return fileManager.documentsDirectory?.appendingPathComponent("Translations.nstack")
+        #if !(os(watchOS))
```

Comment:
Yeah i stashed from the watchOS branch and this is a leftover change will remove it.